02-microarray/clustering_microarray_01_heatmap.Rmd

```diff
@@ -1,145 +1,347 @@
+Clicking this link will most likely send this to your downloads folder on your computer. 
+Move this `.Rmd` file to where you would like this example and its files to be stored.
+
+## Set up your analysis folders 
+
+Good file organization is helpful for keeping your data analysis project on track!
+We have set up some code that will automatically set up a folder structure for you. 
+Run this next chunk to set up your folders! 
 
-Create output folder.
+If you have trouble running this chunk, see our [introduction to using `.Rmd`s](https://alexslemonade.github.io/refinebio-examples/01-getting-started/getting-started.html#how-to-get-and-use-rmds) for more resources and explanations. 
 
 ```{r}
+# Create the data folder if it doesn't exist
+if (!dir.exists("data")) {
+  dir.create("data")
+}
+
 # Define the file path to the plots directory
-plots_dir <- "plots" # Replace with path to desired output plots directory
+plots_dir <- "plots" # Can replace with path to desired output plots directory
 
 # Create the plots folder if it doesn't exist
 if (!dir.exists(plots_dir)) {
   dir.create(plots_dir)
 }
+
+# Define the file path to the results directory
+results_dir <- "results" # Can replace with path to desired output results directory
+
+# Create the results folder if it doesn't exist
+if (!dir.exists(results_dir)) {
+  dir.create(results_dir)
+}
+```
+
+In the same place you put this `.Rmd` file, you should now have three new empty folders called `data`, `plots`, and `results`!
+
+## Obtain the dataset from refine.bio
+
+For general information about downloading data for these examples, see our ['Getting Started' section](https://alexslemonade.github.io/refinebio-examples/01-getting-started/getting-started.html#how-to-get-the-data). 
+
+Go to this [dataset's page on refine.bio](https://www.refine.bio/experiments/GSE24862).
+
+Click the "Download Now" button on the right side of this screen.  
+
+<img src="https://github.com/AlexsLemonade/refinebio-examples/raw/e140face75daa6d2c34e30a4755c362e6039a677/template/screenshots/download-now.png" width=200>  
+
+Fill out the pop up window with your email and our Terms and Conditions:  
+
+<img src="https://github.com/AlexsLemonade/refinebio-examples/raw/e140face75daa6d2c34e30a4755c362e6039a677/template/screenshots/download-email.png" width=500>
+
+It may take a few minutes for the dataset to process.
+You will get an email when it is ready. 
+
+## About the dataset we are using for this example
+
+For this example analysis, we will use this [melanoma cell line dataset](https://www.refine.bio/experiments/GSE24862).
+
+The data that we downloaded from refine.bio for this analysis has 21 microarray samples obtained from three PLX4032-sensitive parental and three PLX4032-resistant sub-lines of melanoma cell lines that were treated or not treated with the RAF-selective inhibitor, PLX4032.
```

Comment:
I think this sentence has a lot of info in it so its a lot to absorb at once. Can we spread it out so it's a bit easier to follow?

Reply:
I broke up this sentence in the last commit, as suggested. I believe that the language here was the easiest way to word what's going on in the dataset while keeping the sentence relatively short. However, if the wording here still seems like a lot, I can attempt to re-word it, which would likely include adding a sentence or two.